### PR TITLE
React uploader

### PR DESCRIPTION
**Pull request by farhoud**
I feel like having 2 app for a gallery is too much
so if @gitaaron don't mind i use the react-gallery.

but don't worry i can move them.

### CODE REVIEW

**Comment by gitaaron** [comment]
Closing this for now since there is still more work to be done to close #59 and #60

**Comment by farhoud** [comment]
added nice features

**Comment by farhoud** [comment]
fix conflicts

File: examples/react-gallery/src/Store.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const Store = (id) => {
```
Comment (gitaaron) [inline examples/react-gallery/src/Store.js:1]: name 'Store' is a little confusing and I think we should abstract this away from user eventually anyway - will pass review for now though and added a new issue for this -

https://github.com/functionland/fula/issues/86

File: libraries/fula-client/src/index.ts [hunk 2]
```diff
@@ -1,165 +1,180 @@
 import type {SchemaProtocol} from "@functionland/file-protocol";
```
Comment (gitaaron) [inline libraries/fula-client/src/index.ts:1]: Changes to the library should go in a separate PR/issue from changes to the samples.

File: libraries/fula-client/src/index.ts [hunk 3]
```diff
@@ -1,165 +1,180 @@
...
+  submitQuery,
+  submitSubscriptionQuery
+} from '@functionland/graph-protocol'
 import {configure} from './config';
 import Libp2p, {constructorOptions, Libp2pOptions} from 'libp2p';
 import {Connection, Status} from "./connection"
+import debug from "debug";
+import PeerId from "peer-id";
+
+debug.disable()
```
Comment (gitaaron) [inline libraries/fula-client/src/index.ts:16]: why disable debug here?

**Comment by gitaaron** [comment]
I am getting the following error when trying to run the sample -

```
Compiled with problems:X

ERROR in ./src/App.js 42:45-58

export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)


ERROR in ./src/App.js 70:22-39

export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)


ERROR in ./src/App.js 73:26-39

export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)


ERROR in ./src/components/ConnInfo.jsx 24:13-26

export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)


ERROR in ./src/components/ConnInfo.jsx 33:13-27

export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)


ERROR in ./src/components/ConnInfo.jsx 42:13-30

export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)
```

**Comment by gitaaron** [comment]
@mehdibalouchi  can you please review the changes to library in here or @farhoud  can you remove it from this PR and create a new one for it?  I'll let you guys figure it out.

File: examples/react-gallery/src/Store.js (hunk 1 above)
Comment (farhoud) [reply examples/react-gallery/src/Store.js:1]: i think i should remove it. i don't use it. it make a sample to complicated

File: libraries/fula-client/src/index.ts (hunk 3 above)
Comment (farhoud) [reply libraries/fula-client/src/index.ts:16]: I have to do it some where because it only manage able throw code or local storage variable and it consume a lot of resource in browser.
but you are right it think we should make an issue to add:
[env-build-blugin](https://www.npmjs.com/package/esbuild-envfile-plugin)
[.env](https://www.npmjs.com/package/dotenv)
and move logger somewhere more meaningfull

**Comment by farhoud** [comment]
> I am getting the following error when trying to run the sample -
> 
> ```
> Compiled with problems:X
> 
> ERROR in ./src/App.js 42:45-58
> 
> export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)
> 
> 
> ERROR in ./src/App.js 70:22-39
> 
> export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)
> 
> 
> ERROR in ./src/App.js 73:26-39
> 
> export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)
> 
> 
> ERROR in ./src/components/ConnInfo.jsx 24:13-26
> 
> export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)
> 
> 
> ERROR in ./src/components/ConnInfo.jsx 33:13-27
> 
> export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)
> 
> 
> ERROR in ./src/components/ConnInfo.jsx 42:13-30
> 
> export 'Status' (imported as 'Status') was not found in '@functionland/fula' (possible exports: createClient)
> ```

fixed it. it was mess i made in merging change's.
